### PR TITLE
Update User Guide and Developer Guide for find feature

### LINKED ISSUE #50: Update DG and UG based on your part
Whatever you change, also update in the UG and DG

### PR DESCRIPTION
**Pull request by yeoutzer**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -331,6 +331,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2a1. FlashBack shows an error message
       
       Use case resumes at step 1
+    
+**Use case: Find cards by search criteria**
+
+**MSS**
+
+1. User requests to find cards by criteria with given keywords
+1. FlashBack shows a list of flashcards matching given keywords according to search criteria
```
Comment (zenlyj) [inline docs/DeveloperGuide.md:366]: Should this be step 2 instead of step 1?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -95,6 +96,23 @@ Format: `view INDEX`
...
 
+### Finding cards : `find`
+
+Find cards according to search criteria containing any of the given keywords.
+
+Format: `find CRITERIA KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g `equation` will match `Equation`
+* The order of the keywords does not matter. e.g. `Newton Equation` will match `Equation Newton`
+* The search will be according to the `CRITERIA` given. `q/` to search by questions or `c/` to search by categories.
```
Comment (zenlyj) [inline docs/UserGuide.md]: Perhaps you should also state that the user can search by specifying both the question and category. eg. "find q/ <keyword> c/ <keyword>".

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yeoutzer) [reply docs/DeveloperGuide.md:366]: It is okay, the documentation will be reflected as step 1 and step 2 correctly.

File: docs/UserGuide.md (hunk 2 above)
Comment (yeoutzer) [reply docs/UserGuide.md]: It is okay, the user will not be able to search by multiple criteria, it will be either by question or by category.

**Pull request merged**